docs/source/release_notes.rst

```diff
@@ -38,6 +38,8 @@ v0.5.0 - (10th December 2024)
 - **Outputs Role Performance Optimization:** Output role handles dict data directly and only converts to DataFrame on Database write.
 - **Overall Performance Optimization:** The overall performance of the framework has been improved by a factor of 5x to 12x
   depending on the size of the simulation (number of units, markets, and time steps).
+- **Learning Opservation Space Scaling:** Instead of the formerly used max sclaing of the observation space, we added a min-max scaling to the observation space.
```

Comment:
since there has been a new release already, these should be moved up to upcoming release